src/java/org/apache/cassandra/tools/nodetool/InvalidateCIDRPermissionsCache.java

```diff
@@ -39,8 +39,8 @@ public void execute(NodeProbe probe)
     {
         if (args.isEmpty())
         {
-            probe.invalidateCidrPermissionsCache("");
-            probe.output().out.println("Invalidated CIDR permissions cache");
+            if(probe.invalidateCidrPermissionsCache(""))
```

Comment:
This change doesn't hurt, but works fine without this change as well. Invalidating entire CIDR permissions cache doesn't go through invalidateCidrPermissions(String roleName) we are changing above. i.e, we always get return value true here. 

```
    @Override
    public boolean invalidateCidrPermissionsCache(String roleName)
    {
        if (roleName == null || roleName.isEmpty())
        {
            cidrPermissionsCache.invalidate();
            return true;
        }

        return cidrPermissionsCache.invalidateCidrPermissions(roleName);
    }
```

Reply:
But `return cidrPermissionsCache.invalidateCidrPermissions(roleName);` can return false? 
```
public boolean invalidateCidrPermissions(String roleName)
    {
        if (cache.getIfPresent(RoleResource.role(roleName)) == null)
            return false;

        invalidate(RoleResource.role(roleName));
        return true;
    }
```

Reply:
Yes, but this line 42 gets executed when args.isEmpty, i.e, user wants to invalidate entire cache, so cidrPermissionsCache.invalidateCidrPermissions(roleName) is not called in this case. cidrPermissionsCache.invalidateCidrPermissions(roleName) is called in the below else condition and we check for return value there - line 49.

Reply:
Oh I see what you mean about the full cache invalidation